.github/workflows/ci.yml

```diff
@@ -87,6 +87,7 @@ jobs:
               - barretenberg/cpp/src/barretenberg/vm/**
               - barretenberg/cpp/src/barretenberg/**/generated/*
               - barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.{hpp,cpp}
+              - barretenberg/cpp/src/barretenberg/bb/main.cpp
```

Comment:
seems obvious to run e2e tests if there is a change to main.cpp...

Reply:
you can add the e2e-all label now to your PR to have it run in CI

Reply:
yeah but don't you agree it should run automatically if we change the single interface to bb?

Reply:
oh whoops, didn't read this change carefully. Yes I agree with this change